packages/bierzo-wallet/src/store/rpcendpoint/actions.ts

```diff
@@ -0,0 +1,16 @@
+import { Action } from "redux";
+import { ActionType } from "typesafe-actions";
+
+import { RpcEndpoint } from "../../communication/rpcEndpoint";
+
+export interface SetRpcEndpointActionType extends Action {
+  type: "@@rpcendpoint/SET_RPC_ENDPOINT";
```

Comment:
This type name is little redundant because you already have `@@rpcendpoint` in type name, so better name will be `@@rpcendpoint/SET`.

Reply:
Done in https://github.com/iov-one/ponferrada/pull/661/commits/6142d8970489eb77e2da0180987d16dff3b63b68